### PR TITLE
fix wiki links

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
originally reported in https://community.theforeman.org/t/broken-link-in-documentation/33957/

File: _includes/manuals/3.5/7.3_getting_help.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Please check the [Troubleshooting wiki page](http://projects.theforeman.org/projects/1/wiki/Troubleshooting) for solutions to the most common problems.  Otherwise, there are two primary methods of getting support for the Foreman: IRC and discussion forums.
+The two primary methods of getting support for the Foreman are IRC and discussion forums.
```
Comment (maximiliankolb) [inline _includes/manuals/3.5/7.3_getting_help.md]:
```suggestion
The two primary methods of getting support for Foreman are IRC and discussion forums.
```

File: _includes/manuals/nightly/7.3_getting_help.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Please check the [Troubleshooting wiki page](http://projects.theforeman.org/projects/1/wiki/Troubleshooting) for solutions to the most common problems.  Otherwise, there are two primary methods of getting support for the Foreman: IRC and discussion forums.
+The two primary methods of getting support for the Foreman are IRC and discussion forums.
```
Comment (maximiliankolb) [inline _includes/manuals/nightly/7.3_getting_help.md]:
```suggestion
The two primary methods of getting support for Foreman are IRC and discussion forums.
```

File: _includes/manuals/3.6/7.3_getting_help.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Please check the [Troubleshooting wiki page](http://projects.theforeman.org/projects/1/wiki/Troubleshooting) for solutions to the most common problems.  Otherwise, there are two primary methods of getting support for the Foreman: IRC and discussion forums.
+The two primary methods of getting support for the Foreman are IRC and discussion forums.
```
Comment (maximiliankolb) [inline _includes/manuals/3.6/7.3_getting_help.md]:
```suggestion
The two primary methods of getting support for Foreman are IRC and discussion forums.
```

File: _includes/manuals/3.7/7.3_getting_help.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Please check the [Troubleshooting wiki page](http://projects.theforeman.org/projects/1/wiki/Troubleshooting) for solutions to the most common problems.  Otherwise, there are two primary methods of getting support for the Foreman: IRC and discussion forums.
+The two primary methods of getting support for the Foreman are IRC and discussion forums.
```
Comment (maximiliankolb) [inline _includes/manuals/3.7/7.3_getting_help.md]:
```suggestion
The two primary methods of getting support for Foreman are IRC and discussion forums.
```

**Review by maximiliankolb — APPROVED** [review]
Thanks Evgeni. One minor suggestion, rest LGTM

**Pull request merged**